src/zeit/retresco/connection.py

```diff
@@ -96,20 +96,8 @@ def get_topicpage_documents(self, id, start=0, rows=25, filter=None):
         response = self._request(
             'GET /topic-pages/{}/documents'.format(id),
             params=params)
-        result = zeit.cms.interfaces.Result()
+        result = zeit.cms.interfaces.Result(response['docs'])
         result.hits = response['num_found']
-        for row in response['docs']:
```

Comment:
IMHO this part shows the essence of the beauty of the new approach quite nicely :-)